mcs/class/System/Mono.UnityTls/X509ChainImplUnityTls.cs

```diff
@@ -45,7 +47,7 @@ internal X509ChainImplUnityTls (UnityTls.unitytls_x509list_ref nativeCertificate
 					elements = new X509ChainElementCollection ();
 					UnityTls.unitytls_errorstate errorState = UnityTls.NativeInterface.unitytls_errorstate_create ();
 					var cert = UnityTls.NativeInterface.unitytls_x509list_get_x509 (nativeCertificateChain, (size_t)0, &errorState);
-					for (int i = 0; cert.handle != UnityTls.NativeInterface.UNITYTLS_INVALID_HANDLE; ++i) {
+					for (int i = 1; cert.handle != UnityTls.NativeInterface.UNITYTLS_INVALID_HANDLE; ++i) {
```

Comment:
What's at index 0 now? Avoiding the duplicate signature?

Reply:
The line above fetches element 0:
`var cert = UnityTls.NativeInterface.unitytls_x509list_get_x509 (nativeCertificateChain, (size_t)0, &errorState);`
Then we get data from that `cert`, and then do `unitytls_x509list_get_x509` with `i`, so before the fix we would have called `unitytls_x509list_get_x509` with 0 twice.

The loop is arguably a bit hard too read (which is why I got this wrong in the first place), so maybe I should just reformulate that thing into a while(true) with transparent break condition?

Reply:
I think a comment would be sufficient to clarify why we're skipping 0. I could see someone coming back and getting curious as to why we're not starting at 0. Just a nitpick I suppose. :-)